programmers-coding-test/sangjun/DFS-BFS/미로탈출.js

```diff
@@ -0,0 +1,46 @@
+const n = 5;
+const m = 6;
+
+const maps = [
+    [1, 0, 1, 0, 1, 0],
+    [1, 1, 1, 1, 1, 1],
+    [0, 0, 0, 0, 0, 1],
+    [1, 1, 1, 1, 1, 1],
+    [1, 1, 1, 1, 1, 1]
+];
+
+// 상하좌우
+let dx = [-1, 1, 0, 0];
+let dy = [0, 0, -1, 1];
+
+const queue = [];
+
+function bfs(x, y) {
+    queue.push([x, y])
+
+    while(queue.length) {
+        const node = queue.shift();
+
+        let x = node[0]
+        let y = node[1]
+
+        for (let i = 0; i < dx.length; i++) {
+            let nx = x + dx[i];
+            let ny = y + dy[i];
```

Comment:
nx, ny를 밑에서 수정하는 일이 없으니 const 가 맞음

```suggestion
            const nx = x + dx[i];
            const ny = y + dy[i];
```